packages/devtools_app/lib/src/service/editor/editor_client.dart

```diff
@@ -212,12 +227,39 @@ class EditorClient extends DisposableController
     );
   }
 
+  /// Gets the editable arguments from the Analysis Server.
+  Future<EditableArgumentsResult?> getEditableArguments({
+    required TextDocument textDocument,
+    required CursorPosition position,
+  }) async {
+    final response = await _callLspApi(
+      LspMethod.editableArguments,
+      params: {
+        'type': 'Object', // This is required by DTD.
+        'textDocument': textDocument.toJson(),
+        'position': position.toJson(),
+      },
+    );
+    final result = response.result['result'];
+    if (result != null) {
+      return EditableArgumentsResult.fromJson(result as Map<String, Object?>);
+    }
+    return null;
```

Comment:
prefer a ternary to make this terse. I thought we had a DCM lint for this but maybe didn't ever enable it.
```
return result != null ? EditableArgumentsResult.fromJson(result as Map<String, Object?>) : null;
```

Reply:
done!